### ISSUE TITLE
[FEATURE] obby farmer module

### ISSUE THREAD

**Issue by Badinek** [John200410/rusherhack-issues#498]
**What feature would you like to be added.**
A detailed explanation of what this feature should be.
it would place and break echests repeatedly- and also 2 other things for it: (copy pasted from discord)

1. compatible with baritone- for example, i have selection set, i turn on the farmer and then i start command to fill the selection, the obby farmer would have priority over the selection- so it would wait until the obby is farmed and then fill the selection

2. "autorefill"- once the selection is done if theres any more echests in obby it would start farming again and then start placing again

oh and also please make it use vanilla mining, packet mining is broken on 2b2t and can cause kicks/getting stuck 

**Is your feature request related to a problem? Please describe.**
not a problem in rusherhack, just that im working in a big project and its very needed since others clients have it but in bad quality

**Comment by John200410** [comment]
Thank you for the suggestion.

**Comment by jumboman32** [comment]
Duplicate of #337

**Comment by Badinek** [comment]
> Duplicate of #337

that one has 0 explanation and mine also contains more stuff so no not really

**Comment by jumboman32** [comment]
https://github.com/John200410/rusherhack-issues/issues/337#issuecomment-759729151
